bin/mozilla/custom_am.pl

```diff
@@ -592,10 +596,13 @@ sub do_dbcheck {
   print qq|<th class=listheading>|.$locale->text('Net Amount').qq|</td>|;
   print qq|<th class=listheading>|.$locale->text('Invoice Tax').qq|</td>|;
   print qq|<th class=listheading>|.$locale->text('Line Tax').qq|</td>|;
+  print qq|<th class=listheading>|.$locale->text('Diff').qq|</td>|;
   print qq|</tr>|;
 
   $i = 0;
 
+  $form->{mindiff} *= 1;
```

Comment:
whatfor do we need a *= 1?

Reply:
This makes sure if the variable is null or non-number, it is converted to 0.

Reply:
ok, thank you for explaining :)